overridable/bool_or_string.go

```diff
@@ -0,0 +1,82 @@
+package overridable
+
+import (
+	"encoding/json"
+)
+
+// BoolOrString is a set of rules that either evaluate to a string or a bool.
+type BoolOrString struct {
+	rules rules
+}
+
+// FromBoolOrString creates a BoolOrString representing a static, scalar value.
+func FromBoolOrString(v interface{}) BoolOrString {
+	return BoolOrString{
+		rules: rules{SimpleRule(v)},
+	}
+}
+
+// Value returns the PublishedValue for the given repository.
```

Comment:
This might be a bit too specific now:

```suggestion
// Value returns the value for the given repository.
```